x/auth/client/rest/query.go

```diff
@@ -64,7 +71,12 @@ func QueryAccountRequestHandlerFn(
 			return
 		}
 
-		rest.PostProcessResponse(w, cliCtx, account)
+		if cliCtx.Height > 0 {
```

Comment:
I don't think this is the correct logic exactly. What we want rather is to always set the height via: `cliCtx.Node().Height(nil)`

Reply:
isn't this done already with `rest.ParseQueryHeightOrReturnBadRequest(w, cliCtx, r)`?

Reply:
No no @fedekunze. That parses a height query from a client request. What we're trying to do here is return the _current_ height with the account when the account is queried for.

Reply:
ohhh got it, sorry completely misunderstood the goal of the PR